library/src/main/java/com/devbrackets/android/exomedia/EMVideoView.java

```diff
@@ -82,7 +84,7 @@
     private ImageView previewImageView;
 
     private TouchVideoView videoView;
-    private VideoSurfaceView exoVideoSurfaceView;
+    private VideoTextureView mExoVideoTextureView;
```

Comment:
Please don't use Hungarian notation. (this variable should be `exoVideoTextureView` not `mExoVideoTextureView`